openlibrary/templates/borrow_admin.html

```diff
@@ -341,75 +335,3 @@ <h4>Waiting List JSON</h4>
         <pre>$:json_encode([doc.dict() for doc in waitinglist], indent=True)</pre>
     </div>
 </div>
-
-
-<style type="text/css">
-body#plain #borrowTable {margin: 30px 0 25px;}
-body#plain #borrowTable table {width:100%;font-family:"Lucida Grande","Arial",sans-serif;}
-body#plain #borrowTable table th {
-    border-bottom: 1px solid #c7c3b3;
-    vertical-align: bottom;
-    padding-bottom:10px;
-    font-family:"News Gothic MT","Trebuchet MS",Geneva,Helvetica,sans-serif;
-    color:#615132;
-    font-weight:600;
-    font-size:0.6875em;
-    text-transform: uppercase;
-    white-space: nowrap;
-    text-align: center;
-}
-
-th.xtitles {
```

Comment:
This rule appears to be lost. Was this intentional?

Reply:
&.titles {
        text-align: left;
      }
I thought that "x" was typo.

Reply:
https://github.com/internetarchive/openlibrary/pull/4482#diff-ebde383b40e87da92da077076dab958e61e8bedb93d959b16d5d1501234dfb8dR25

Reply:
appears to be used on L102 and L166